app/experimenter/docs/adrs/0008-integrating-htmx.md

```diff
@@ -0,0 +1,78 @@
+# Front-end web development with HTMX
+
+- Status: accepted
+- Deciders: Yashika Khurana, Jared Lockhart, Elise Richards
+- Date: 2022-10-25
+
+## Context and Problem Statement
+
+Nimbus UI uses GraphQL to fetch data from the backend and it highly depends on the three packages i.e. [graphene-django](https://github.com/graphql-python/graphene-django), [graphene](https://github.com/graphql-python/graphene) and [apollo-client](https://github.com/apollographql/apollo-client). We have found out that using GraphQL significantly hinders the performance of the Experimenter as it adds extra overhead while making the call and the benefit of using GraphQL is not something we are making use of. We also have to significantly maintain the burden to implement most of the API in both DRF (Django Rest Framework) and GraphQL, and additional testing and tools are required to support that. As our application is growing so what steps we should take to improve the performance and better maintainability of the application?
```

Comment:
>highly depends on the three packages

Can you add info here that explains why this is a problem? I know there is some info below but I think it would be valuable to have it upfront.